Alignment/OfflineValidation/plugins/PrimaryVertexValidation.cc

```diff
@@ -2184,6 +2381,41 @@ void PrimaryVertexValidation::fillTrendPlot(TH1F* trendPlot, TH1F* residualsPlot
 }
 
 //*************************************************************
+void PrimaryVertexValidation::fillTrendPlotByIndex(TH1F* trendPlot,std::map<unsigned int, TH1*>& h, TString fitPar_, TString var_)
```

Comment:
please change this interface to avoid strings (let alone TStrings). It looks like enums would serve the same purpose and avoid many string comparisons.